Automattic/woocommerce-payments
# Use bespoke prefix for identifiers in browser environment

Fixes https://github.com/Automattic/woocommerce-payments/issues/165

#### Changes proposed in this Pull Request

Refactor to use `wcpay` prefix for:
- DOM IDs
- DOM classes
- DOM input names
- Asset files
- JS config variable

#### Testing instructions



* Verify no behavior change: the payment flow should work just as before

-------------------
